lib/json_schemer/configuration.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module JSONSchemer
+  class Configuration
+    module Defaults
```

Comment:
Dropped `DEFAULT_` from the constant names since I moved them within this `Defaults` module